### ISSUE TITLE
second time create event not show APPROVECREATEEVENT in activities history

### ISSUE THREAD

**Issue by henryzbw** [bodhiproject/bodhi-ui#404]
The second time I create an event, `APPROVECREATEEVENT` doesn't show in `activities history`
checked db, it has `APPROVECREATEEVENT`, so it's probably a UI bug
![image](https://user-images.githubusercontent.com/10339131/37934479-2f4eddcc-3103-11e8-9805-4d06b4a571c2.png)

**Comment by henryzbw** [comment]
checked via localhost:5555/graphiql
do query 
```
{
  allTransactions(filter: {OR: [{type: APPROVECREATEEVENT}]}) {
    name
  }
}
```
Has the corresponding `APPROVECREATEEVENT`

**Comment by frankobe** [comment]
@dwalintukan @liviavinci Diagnose this with @henryzbw, UI doesnt update this table as expected.

**Comment by dwalintukan** [comment]
@henryzbw might be fixed #418 . please test and verify.

**Comment by henryzbw** [comment]
@dwalintukan fixed
